cmd/lakefs/cmd/run.go

```diff
@@ -199,6 +199,11 @@ const runBanner = `
 │     check out the docs at https://docs.lakefs.io/quickstart/repository
 │
 
+│
+│ If you have any questions or need technical support,
+│     join our Slack channel https://join.slack.com/t/lakefs/shared_invite/zt-g86mkroy-186GzaxR4xOar1i1Us0bzw
```

Comment:
If I remember correctly from @YaelRiv these links might change (if you forget to renew them after a given amount of time you can create a new one but it'll be different). This is happened to us once before.

Given we can't trust all users to use the most current version of lakeFS, I think this should point to a URL we control, that will redirect to whatever the current Slack invite URL is. It's a bit more work but since the link is frozen for a given lakeFS version, this has to resolve to something that works.

Reply:
Yikes!
I think I found a [way](https://getlowdown.com/how-to-create-a-slack-community-and-landing-page-in-one-hour/) (Get your invite link section) 
 to get a permanent link, who's the admin of the public workspace and can try it out? @ozkatz @YaelRiv

Reply:
Now the link never expires :)